lib/config.rb

```diff
@@ -25,9 +25,13 @@ def define_entries
       default_config_file(Machinery::DEFAULT_CONFIG_FILE)
 
       entry("hints",
-        default:     true,
+        default: true,
         description: "Show hints about usage of Machinery in the context of the commands ran by the user"
       )
+      entry("experimental-features",
+        default: false,
+        description: "Enable experimental features. Use at your own risk."
```

Comment:
Would a link to the wiki page make sense here? It's not that I'm requesting such a link, but I wonder if that could increase usability.